src/app/_components/MarkdownContent.tsx

```diff
@@ -14,7 +18,14 @@ export function MarkdownContent({ children }: MarkdownContentProps) {
       className="prose"
       components={{
         img: ({ src, alt }) => (
-          <CustomImage src={src as string} alt={alt as string} />
+          <DynamicImage
+            src={src || ''}
+            alt={alt || ''}
+            width={800}
+            height={450}
```

Comment:
These don't affect the image size right? Are we passing the values just because its required?

Reply:
Exactly! I explained why in the PR description above:

This `DynamicImage` instance uses absolute `width` and `height` values instead of `fill` with a relatively positioned parent. This is because the second option creates a `p` > `div` > `img` HTML tree, which throws a hydration error as a `div` cannot be the direct descendant of a paragraph.

We pass absolute values because we have to, but the image keeps its original aspect ratio.